server/src/controllers/GameControllers/game.js

```diff
@@ -0,0 +1,112 @@
+import Snake from './snake';
+import SNAKE_CONSTANTS from './gameConstants';
+
+class Game {
+  constructor(playerOne, socketOne, io) {
+    this.roomId = (Math.random() * 100000000000).toFixed(0);
+    this.createSocketEvents(playerOne, socketOne);
+    const initialProperties = {
+      head: { x: 600, y: 200 },
+      direction: 'down',
+      length: SNAKE_CONSTANTS.INITIAL_LENGTH,
+    };
+    this.firstSnake = new Snake(playerOne, initialProperties, this);
+    this.secondSnake = null;
+    this.freeToJoin = true;
+    this.winner = null;
+    this.ended = false;
+    this.io = io;
+  }
+
+  joinGame(PlayerTwo, socketTwo) {
```

Comment:
P capital?

Reply:
the old devil typo. I will take care of that.